src/Tutor.Core/LearnerModel/ILearnerService.cs

```diff
@@ -5,5 +5,6 @@ namespace Tutor.Core.LearnerModel
     public interface ILearnerService
     {
         Result<Learner> Register(Learner learner);
+        Result<Learner> GetInformations(int id);
```

Comment:
Information doesn't have a plural form and it is a generic word :)

GetLearnerProfile?